azure-iot-device/tests/common/test_mqtt_transport.py

```diff
@@ -299,6 +424,29 @@ def test_calls_event_handler_callback_externally_driven(
         assert callback.call_count == 1
         assert callback.call_args == mocker.call()
 
+    @pytest.mark.parametrize(
+        "error_params",
+        conack_return_codes,
+        ids=["{}->{}".format(x["name"], x["error"].__name__) for x in conack_return_codes],
+    )
+    @pytest.mark.it(
+        "Triggers on_mqtt_connection_failure event handler callback upon completion of externally-driven reconnect returns failed RC"
```

Comment:
>reconnect [](start = 105, length = 9)

If this is for reconnect, shouldn't it be in the reconnect tests?

Also, I'm not clear what an externally driven reconnect is

Reply:
brain fart.

---
In reply to: [307912760](https://github.com/Azure/azure-iot-sdk-python-preview/pull/136#discussion_r307912760) [](ancestors = 307912760)